website/docs/sdk-reference/dart.md

```diff
@@ -130,34 +158,68 @@ final user = ConfigCatUser(
 );
 ```
 
-## Polling Modes
-The *ConfigCat SDK* supports 3 different polling mechanisms to acquire the setting values from *ConfigCat*. After latest setting values are downloaded, they are stored in the internal cache then all `getValue()` calls are served from there. With the following polling modes, you can customize the SDK to best fit to your application's lifecycle.  
-[More about polling modes.](/advanced/caching/)
+### Default user
 
-### Auto polling (default)
-The *ConfigCat SDK* downloads the latest values and stores them automatically every 60 seconds.
+There's an option to set a default user object that will be used at feature flag and setting evaluation. It can be useful when your application has usually a single user only, or rarely switches users.
```

Comment:
I personally, would remove the word "usually".

Reply:
✅